src/predictors/ReLU.jl

```diff
@@ -314,6 +323,11 @@ function add_predictor(
     z = JuMP.@variable(model, [1:m], base_name = "moai_z")
     _set_bounds_if_finite.(Ref(cons), z, 0, max.(0, -first.(bounds)))
     append!(cons, JuMP.@constraint(model, x .== y - z))
-    append!(cons, JuMP.@constraint(model, y .* z .== 0))
+    if predictor.relax_equality
+        ϵ = predictor.relaxation_parameter
+        append!(cons, JuMP.@constraint(model, y .* z .<= ϵ))
+    else
+        append!(cons, JuMP.@constraint(model, y .* z .== 0))
```

Comment:
What happens if you make this always `<= 0`? Both `y` and `z` are non-negative, correct? Then we could have just a single relaxation parameter.

Reply:
Yes, but solvers behave differently when we change the constraint. Annoyingly, I also have an example where `== 0` works but `<= 0` does not.